### ISSUE TITLE
Useless link to email verification 

### ISSUE THREAD

**Issue by quaestor** [theCrag/website#2948]
**What happened?**
On some pages, the link to the email verification process still appears after having verified the email address already.

Example pages include: 
* Reordering routes
* Editing ascents
* Editing routes
* Create topo
* Uploading photos

<img width="1024" alt="screenshot 2017-11-08 12 21 05" src="https://user-images.githubusercontent.com/54371/32546513-aedecf8e-c47f-11e7-9627-03e51d24bfeb.png">

**Comment by Mdemaillard** [comment]
I believe that menu was removed. I checked all the mentioned pages:
Reorder routes
Editing ascents
Editing routes
Create topo (Add topo from list view)
Upload photo

Closing the issue.

**Comment by quaestor** [comment]
No, this has not been resolved. I still see it e.g. while editing ascents.

**Comment by Mdemaillard** [comment]
correct, i was looking for the old action menu, but now it is behind the user portrait menu
